### DISCUSSION TITLE
Configuring updates

### DISCUSSION THREAD

**Discussion by amiranagram** [nklayman/vue-cli-plugin-electron-builder#1312] [Q&A] [answered]
Hi, a web-dev here, learning electron. How can I tell the builder to generate .nupkg files? Also is there no updater? I have configured an update server and currently I'm stuck on the last step. [See here](https://nuts.gitbook.com/update-windows.html).

```
Your application just need to configure Update.exe or Squirrel.Windows
```

Also this is the code in my `background.js` where it checks for updates.

```
require('update-electron-app')()

const server = 'https://gynops-updater.herokuapp.com/'
const url = `${server}/update/${process.platform}/${app.getVersion()}`

autoUpdater.setFeedURL({ url })

setInterval(() => {
    autoUpdater.checkForUpdates()
}, 60000)

autoUpdater.on('update-downloaded', (event, releaseNotes, releaseName) => {
    const dialogOpts = {
        type: 'info',
        buttons: ['Restart', 'Later'],
        title: 'Application Update',
        message: process.platform === 'win32' ? releaseNotes : releaseName,
        detail: 'A new version has been downloaded. Restart the application to apply the updates.'
    }

    dialog.showMessageBox(dialogOpts).then((returnValue) => {
        if (returnValue.response === 0) autoUpdater.quitAndInstall()
    })
})
```

Help is appreciated.

**Comment by nklayman** [comment, marked as answer]
Since this plugin uses electron-builder to package the application, you should use their updater. Follow [this guide](https://nklayman.github.io/vue-cli-plugin-electron-builder/guide/recipes.html#auto-update) to set it up with your project. The updater you are trying to use appears to be for a different packaging method.
